### DISCUSSION TITLE
SMTP Permit Senders

### DISCUSSION THREAD

**Discussion by wrodriguez-sbi** [postalserver/postal#2697] [Help with using Postal]
Hello,

I'm totally a newbie. I think I configured everything correctly. All of my DNS records check out, I'm able to send an outgoing email just fine. I stood up postal so we have an outgoing smtp server relay for all of our alerts. On our Dell OME, I'm trying to send an email and point to postal but it keeps erroring out. I'm used to adding allow senders for other smtp servers to allow relay, do I need to do the same for postal? If so, where do I add permitted senders? Thanks!

**Comment by willpower232** [comment]
It depends what error message you're seeing?

**Reply by wrodriguez-sbi** [reply]
From Dell OpenManage, its very generic as shown in the first screenshot. On the second screenshot, its from our PowerStore. According to DellEMC, their workaround is to "add a rule to the SMTP server to accept a message without a date."

**Dell OpenManage**
![image](https://github.com/postalserver/postal/assets/135144436/1724f1f7-03ba-412f-8775-ea7e45c60e0b)

**Dell PowerStore**
![image](https://github.com/postalserver/postal/assets/135144436/c15745c3-e92d-460b-a038-1216ef1d69aa)

**Reply by willpower232** [reply]
ah right, you might want to try `postal logs smtp` and see what Postal replied to them with, the error message should make more sense there

**Reply by wrodriguez-sbi** [reply]
According to the logs, I'm seeing "530 Authentication required." Cross-referencing the error, I came across https://github.com/postalserver/postal/issues/73, not sure if my issue is related. I do not have any routes configured.

postal-smtp-1  | [9] [2023-11-14T13:50:06.478] DEBUG -- : [KS1NFW] <= MAIL FROM:<OpenManageEnterprise@email>
postal-smtp-1  | [9] [2023-11-14T13:50:06.478] DEBUG -- : [KS1NFW] => 250 OK
postal-smtp-1  | [9] [2023-11-14T13:50:06.478] DEBUG -- : [KS1NFW] <= RCPT TO:<_email_>
postal-smtp-1  | [9] [2023-11-14T13:50:06.483] DEBUG -- : [KS1NFW] => 530 Authentication required
postal-smtp-1  | [9] [2023-11-14T13:50:06.484] DEBUG -- : [KS1NFW] <= RSET
postal-smtp-1  | [9] [2023-11-14T13:50:06.484] DEBUG -- : [KS1NFW] => 250 OK
postal-smtp-1  | [9] [2023-11-14T13:56:56.149] DEBUG -- : [JBVROJ]    Connection opened from <omit>
postal-smtp-1  | [9] [2023-11-14T13:56:56.149] DEBUG -- : [JBVROJ]    Client identified as <omit>
postal-smtp-1  | [9] [2023-11-14T13:56:56.150] DEBUG -- : [JBVROJ] <= EHLO APX00222919008-B
postal-smtp-1  | [9] [2023-11-14T13:56:56.191] DEBUG -- : [JBVROJ] => 250-My capabilities are
postal-smtp-1  | [9] [2023-11-14T13:56:56.191] DEBUG -- : [JBVROJ] => 250 AUTH CRAM-MD5 PLAIN LOGIN
postal-smtp-1  | [9] [2023-11-14T13:56:56.192] DEBUG -- : [JBVROJ] <= MAIL FROM:<powerstore@email>
postal-smtp-1  | [9] [2023-11-14T13:56:56.192] DEBUG -- : [JBVROJ] => 250 OK
postal-smtp-1  | [9] [2023-11-14T13:56:56.193] DEBUG -- : [JBVROJ] <= RCPT TO:<_email_>
postal-smtp-1  | [9] [2023-11-14T13:56:56.197] DEBUG -- : [JBVROJ] => 530 Authentication required
postal-smtp-1  | [9] [2023-11-14T13:56:56.197] DEBUG -- : [JBVROJ] <= RSET
postal-smtp-1  | [9] [2023-11-14T13:56:56.197] DEBUG -- : [JBVROJ] => 250 OK
postal-smtp-1  | [9] [2023-11-14T13:56:56.198] DEBUG -- : [JBVROJ] <= QUIT
postal-smtp-1  | [9] [2023-11-14T13:56:56.198] DEBUG -- : [JBVROJ] => 221 Closing Connection
postal-smtp-1  | [9] [2023-11-14T13:56:56.198] DEBUG -- : [JBVROJ]    Connection closed

**Reply by willpower232** [reply]
Presumably you need to tick the checkbox for Enable Authentication and provide an SMTP key for the domain you're trying to use.

**Reply by wrodriguez-sbi** [reply]
I don't want to use any authentication, just supply the smtp server, port number, and from email address. Not all of our applications support email authentication. I just want a simple outbound email only smtp server. Is authentication a requirement?

**Reply by willpower232** [reply]
If the device is capable of sending email on its own, you could have a go with inbound routes to forward the message and that wouldn't need authentication.

Otherwise I'm afraid Postal requires authentication, you would have to have something proxying to Postal to add in the authentication bits if you wanted to keep using it.

**Reply by wrodriguez-sbi** [reply]
Our devices require an smtp server/port, but we need postal to relay the email. So, is this where I would configure the incoming route? Not sure I understand how this works, is there an article/video I can review?

![image](https://github.com/postalserver/postal/assets/135144436/743e16c1-19aa-44cd-bb72-2ebd5bb80eb9)

**Reply by willpower232** [reply]
Endpoints are where you want the messages to end up, you could tell Postal that all emails received at whatever@domain.com should be passed off to the endpoint you set up and then use a less secure proxy to email the message to Postal.

**Reply by wrodriguez-sbi** [reply]
Thank you @willpower232  - I'm going to research other options.
